src/react-apps/package.json

```diff
@@ -3,7 +3,8 @@
   "private": true,
   "scripts": {
     "install-deps": "lerna bootstrap --hoist",
-    "test": "jest"
+    "test": "jest",
+    "installz": "lerna exec -- npm i"
```

Comment:
Humorous, I'll allow it. (But should probably be renamed though).

Reply:
renamed to "install-all".